app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job working on the devise setup but I think it would be great if you add `:confirmable` so when the user signup he needs to confirm his email first since after signing up it redirct me to the user index page and this not should be this case the user need to confirm his email first then login with the mail and password 🔥♻💪🏻 so kindly add this as it is mentioned in the requirements 🔥👏🏻

![image](https://github.com/Ochiengsteven/art-post/assets/58553711/58f092a3-1331-408f-8933-8ff63481bc9e)